wiki/Beatmapping/Star_Rating/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+    - SR
+---
+
+# Star Rating
+<!--TODO: Include SR calculations-->
+Star rating, or commonly abbreviated as *SR*, describes a beatmap's difficulty. This number can be seen on the song selection screen or on a beatmap's info page. On the website, beatmaps are automatically given a difficulty icon based on star rating. See the overview:
```

Comment:
should bold the first usage of the title topic, also it's so common that these have other names/abbreviations, you can put those in parenthesis instead of "or commonly abbreviated as ..."

so beginning here would be `**Star rating** (***SR***) describes ...`